bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh

```diff
@@ -0,0 +1,10 @@
+gst-launch-1.0 -v \
+      filesrc location=cat.png ! decodebin ! videoconvert ! videoscale ! imagefreeze !\
+      video/x-raw,format=RGB,width=257,height=257,framerate=10/1 ! tee name=t \
+      t. ! tensor_converter !\
+      tensor_transform mode=arithmetic option=typecast:float32,add:0.0,div:255.0 !\
+      tensor_filter framework=tensorflow-lite model=tflite_model/deeplabv3_257_mv_gpu.tflite !\
+      tensor_decoder mode=image_segment option1=tflite !\
```

Comment:
Your decoder could not handle the out of all the image segment tflite models. Need to consider it. In my opinion, you do not need option1 so far.

Reply:
Or tflite/image-segment/$model_name?

I am not sure....